sdk/communication/Azure.Communication.Administration/CHANGELOG.md

```diff
@@ -1,8 +1,31 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
```

Comment:
nit: add empty line between ## and ###

At least my VS Code had actually complains whenever I forget that.

Reply:
Thanks @DominikMe Added the extra line